### PR TITLE
Fix use after free

### PR DESCRIPTION
**Pull request by AeroNotix**

### CODE REVIEW

**Comment by AeroNotix** [comment]
This kept outputting garbage on my system, dug into it and found that. `g_error` closes the program so there's no need to free anything.

File: erln8.c [hunk 1]
```diff
@@ -1444,8 +1444,7 @@ int main(int argc, char* argv[]) {
     if(!result) {
       g_hash_table_destroy(erlangs);
       g_hash_table_destroy(runtime_options);
-      g_free(s);
-      g_error("Can't run %s, check to see if the file exists\n", s);
+      g_error("Can't run %s, hello check to see if the file exists\n", s);
```
Comment (metadave) [inline erln8.c]: thanks for the patch, can you take out `hello` from this string? Thanks!

Comment (metadave) [inline erln8.c]: I believe the correct fix is to move the `g_error` line before the `g_free`

Comment (AeroNotix) [inline erln8.c]: The `hello` is some debugging I added and left in accidentally, I will remove it.

Isn't the `g_error` this function: https://developer.gnome.org/glib/stable/glib-Message-Logging.html#g-error ?

Comment (metadave) [inline erln8.c]: ah, yes, good point. It's been a few years since I've looked at glib.

Comment (metadave) [inline erln8.c]: merged as-is, I removed `hello` and pushed another commit. Thanks again!

**Comment by AeroNotix** [comment]
Cheers! Sorry I got caught up in day job. Thanks for merging.

**Pull request merged**